howto/install-appliance/finalise.md

```diff
@@ -1,6 +1,20 @@
 After installing the Anbox Cloud Appliance, you must initialise it and register your Ubuntu SSO account with the dashboard.
 
 <a name="initialise"></a>
```

Comment:
The anchor should stay with the "Initialise" title below.

Reply:
Fixed